doc/layout-config-reference.rst

```diff
@@ -53,9 +57,10 @@ options:
    A partition label.
 
 ``type`` (string)
-   The partition type. May be one of ``primary`` or ``logical``. Note, that with
-   the first occurrence of a logical partition the following ones must be
-   logical, too.
+   The partition type. May be one of ``primary`` or ``logical``. Note, that
```

Comment:
This line seems to be unchanged apart from the wrapped `with` at the end. Did you make sure to set the text width to 80 characters in your editor?

Quick tip: You can use `gq` in Vim to autowrap lines. E.g. `gqq` to wrap the current line, `gqj` to wrap the current and below line, `gq` on a visual selection and so on.

Reply:
Fixed. My text width was set to 72.